include/boost/geometry/strategies/spherical/area_huiller.hpp

```diff
@@ -165,13 +217,57 @@ public :
             {
                 excess = -excess;
             }
+/*
+            // Keep track whenever a segment crosses the prime meridian
+            // First normalize to [0,360)
+            //TODO: compress with trapezoidal strategy
+
+            calculation_type p1_lon = geometry::get_as_radian<0>(p1)
+                                    - ( floor( geometry::get_as_radian<0>(p1) / two_pi ) * two_pi );
+            calculation_type p2_lon = geometry::get_as_radian<0>(p2)
+                                    - ( floor( geometry::get_as_radian<0>(p2) / two_pi ) * two_pi );
+
+            calculation_type max_lon = std::max(p1_lon, p2_lon);
+            calculation_type min_lon = std::min(p1_lon, p2_lon);
+
+            if(max_lon > pi && min_lon < pi && max_lon - min_lon > pi)
+            {
+                state.crosses_prime_meridian++;
+            }
+
+            //Test if the segment has a vertex that is the South pole
+            if (geometry::get_as_radian<1>(p1) == - half_pi
+                || geometry::get_as_radian<1>(p2) == - half_pi)
+            {
+                state.south_vertex = true;
+            }
+
+            // Global max, min
+            state.max_lon = std::max(state.max_lon, max_lon);
+            state.min_lon = std::min(state.min_lon, min_lon);
 
+
+            // Test if the segment has a point on northern hemisphere
+            if(state.south)
+            {
+                if(get<1>(p1) > 0 || get<1>(p2) > 0)
+                {
+                    state.south = false;
+                }
+            }
+*/
```

Comment:
Both :) It is work in progress but it is not needed if we decide to get rid of l'Hulier's formula.